teeio-validator/include/ide_test.h

```diff
@@ -123,6 +123,8 @@ typedef enum {
 #define MAX_TEST_CASE_NUM \
   (MAX(IDE_COMMON_TEST_CASE_NUM, CXL_MEM_IDE_TEST_CASE_NUM))
 
+#define TEEIO_KEY_GEN_FIXED_KEY 0x11
```

Comment:
what is this???

Reply:
It seems fix key value. I recommend to use name `TEEIO_TEST_FIXED_TX_KEY_BYTE_VALUE` and `TEEIO_TEST_FIXED_RX_KEY_BYTE_VALUE`.

Reply:
Will update.